README.md

```diff
@@ -95,11 +95,11 @@ Use the button in our [header](https://github.com/amundsen-io/amundsen#readme) t
 
 ## Getting Started
 
-Please visit the Amundsen installation documentation for a [quick start](./docs/installation.md) to bootstrap a default version of Amundsen with dummy data.
+Please visit the Amundsen installation documentation for a [quick start](./installation) to bootstrap a default version of Amundsen with dummy data.
```

Comment:
not sure if it is the correct fix to remove md, when I click the link, it still shows as broken. Correct link should be https://github.com/amundsen-io/amundsen/blob/main/docs/installation.md.

I think the doc website issue is because how our MKDOC config setups changes.